modules/special-rules/pages/deep-strike.adoc

```diff
@@ -0,0 +1,20 @@
+= Deep Strike
+
+`Unit ability`
+
+These units are capable of rapid insertion onto the battlefield via Drop Pods, Teleportation, and other methods.
+
+---
+
+Where all units in a detachment have the {deep-strike} ability:
+
+* The detachment can deploy using the Drop Pods rule, regardless of scenario.
+
+[NOTE]
+====
+This ability is used where a limited number of units in a detachment are capable of arriving via the Drop Pods rule, for example Swooping Hawks in an Eldar Warhost, or Crisis Battlesuits in a Tau Hunter Cadre.
+====
+
```

Comment:
I think I see what you're trying to say, but it may be slightly confusing — I had to do a double-take to parse it after the earlier statement.

Ultimately, if we need to allude to example troop types then I think it's better to do this in the 'fluff' description rather than insert it as a note in the rules. However, if you feel that we ought to expand on the way that Deep Strike units appear alongside non- Deep Strike units in detachment lists, and how you can't mix if you want to use Deep Strike, then I think we should phrase this a different way that is more clear.

```suggestion
```

Reply:
I thought we'd used similar references to example troop types before, but I can't find an example right at the moment.

Reply:
> I thought we'd used similar references to example troop types before, but I can't find an example right at the moment.

It wouldn't be the first time that more than one approach crept into the text for one reason or another, or no reason at all. It was the double-take that first gave me pause for thought — and then I remembered instances where we put this type of example (which applies to the whole thing) up top, and the fact that as a bonus it also helps to declutter the actual rules a bit. As in the Hero ability for example:

![image](https://user-images.githubusercontent.com/34416576/200663983-ea27d749-f946-4980-848d-8469e7e74a10.png)
_(Yes, we/I capitalised Infantry and Vehicle here; standard prose should be _default_ for the descriptive section, but where we switch over from 'fluff' to rules-related 'guidance' then it makes sense to use 'rules speak'.)_

I think I rephrased the lead-in sentence a bit since I first did that double-take; but I think that my suggestion here still makes sense — both for the above reasons, and because the text still presents a slightly mixed message at first glance: That is, we have "*every* unit in your detachment" closely followed by "limited number of units in a detachment", and my mind initially read that as an incongruency (although I accept that it might just be me that is sensitive to this).

I'm not wedded to it, but this is my reasoning.

Reply:
As at https://github.com/GameBrains/er-core/pull/36/files#r1111105849, we won't need this note so I'll apply the earlier suggestion to remove it.